CMakeLists.txt

```diff
@@ -47,7 +47,7 @@ set(SOLVER_CG_EPSILON 1.0e-18)
 set(SOLVER_CG_BETA_TYPE 2)
 
 # type of convergenve for Samurai CG(1 = ~step size(orig), 2 = ||g(X)||/||g(X0)||)
-set(SOLVER_CG_CONV_TYPE 1)
+set(SOLVER_CG_CONV_TYPE 2)
```

Comment:
Is there a reason to change the convergence type here?

Reply:
The previous setting of convergence type is the original value before the code was modified by ASAP.  I don't know how it got set to 1 but the more numerical proper value is setting to be 2.  Allison kept the old option in there but it is really not numerically correct.  I suspect that changing this will modify iteration count, though I have not tested it yet.

Reply:
Thanks for the clarification. Since the `SOLVER_SAMURAI` is currently set to TN solver, I assume this change won't affect the results at all unless we change the solver type?

Reply:
Good point, I missed that this only applies to the CG solver.  Now I understand why it was set to 1 originally.